net/url/test.cc

```diff
@@ -35,37 +35,37 @@ TEST(URLTest, SmokeTest) {
   u = URL("www.google.com");
   EXPECT_EQ("www.google.com", u.host);
   EXPECT_EQ("/", u.path);
-  EXPECT_EQ("http", u.protocol);
+  EXPECT_EQ("http", u.scheme);
   EXPECT_EQ(80, u.port);
 
   u = URL("www.google.com/test");
   EXPECT_EQ("www.google.com", u.host);
   EXPECT_EQ("/test", u.path);
-  EXPECT_EQ("http", u.protocol);
+  EXPECT_EQ("http", u.scheme);
   EXPECT_EQ(80, u.port);
 
   u = URL("www.google.com:8080");
   EXPECT_EQ("www.google.com", u.host);
   EXPECT_EQ("/", u.path);
-  EXPECT_EQ("http", u.protocol);
+  EXPECT_EQ("http", u.scheme);
   EXPECT_EQ(8080, u.port);
 
   u = URL("meh://www.google.com:27960");
   EXPECT_EQ("www.google.com", u.host);
   EXPECT_EQ("/", u.path);
-  EXPECT_EQ("meh", u.protocol);
+  EXPECT_EQ("meh", u.scheme);
   EXPECT_EQ(27960, u.port);
 
   u = URL("meh://www.google.com:27960/bazinga");
   EXPECT_EQ("www.google.com", u.host);
   EXPECT_EQ("/bazinga", u.path);
-  EXPECT_EQ("meh", u.protocol);
+  EXPECT_EQ("meh", u.scheme);
   EXPECT_EQ(27960, u.port);
 
   u = URL("localhost:/test");
```

Comment:
Added!
ᐧ

Thanks,
Dima

http://dimakorolev.com/ | http://www.linkedin.com/in/dimakorolev |
dmitry.korolev@gmail.com | +1 (312) 593 2783


On Thu, Jan 8, 2015 at 12:06 AM, John Babak <notifications@github.com>
wrote:

> In net/url/test.cc
> <https://github.com/KnowSheet/Bricks/pull/36#discussion-diff-22643573>:
>
> >    EXPECT_EQ(27960, u.port);
> >
> >    u = URL("meh://www.google.com:27960/bazinga");
> >    EXPECT_EQ("www.google.com", u.host);
> >    EXPECT_EQ("/bazinga", u.path);
> > -  EXPECT_EQ("meh", u.protocol);
> > +  EXPECT_EQ("meh", u.scheme);
> >    EXPECT_EQ(27960, u.port);
> >
> >    u = URL("localhost:/test");
>
> I meant without the path ("test") but with the slash. Either
> URL("localhost/") or URL("localhost:/"), the colon is tested in your
> test, the path with the only slash isn't.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KnowSheet/Bricks/pull/36/files#r22643573>.
>